Header header

int64 numOfClusters
float64 minDistance